content/fr/multitenancy.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Mutualisation (Multitenancy)
+status: Completed
+category: Property
+tags: ["architecture", "property", ""]
+---
+
+La mutualisation (multitenancy en anglais) fait référence à une installation unique de logiciel utilisé par plusieurs locataires (tenant).
+Un locataire est un utilisateur, une application ou un groupe d'utilisateurs/applications qui utilisent le logiciel pour travailler sur ses propres jeux de données.
+Ces locataires ne partagent pas de données (sauf si c'est explicitement demandé par le propriétaire) et peuvent même ne pas se connaître.
+
+Un locataire peut être petit comme un utilisateur indépendant avec un unique identifiant de connexion 
+— Penser à un logiciel de productivité personnelle — ou aussi grand qu'une entreprise complète avec 
+des centaines d'identifiants de connexion, chacun avec ses propres droits mais interdépendants de multiples
+manières. Les exemples de colocation incluent : Google Mail, Google Docs, Microsoft Office 365, Salesforce CRM,
+Dropbox, parmi tant d'autres qui sont complètement ou partiellement mutualisé.
+
+## Problème auquel il répond
+
+Sans mutualisation, chaque locataire aurait besoin d'une installation logicielle qui lui est dédiée.
+Cela augmente l'utilisation de ressources et l'effort de maintenance, et au final le coût du logiciel.
+
+## Quel en est l'utilité
+
+Le logiciel mutualisé fournit pour chaque locataire un environnement ségrégué (données de travail, paramètres, liste d'identifiants, etc.),
+servant simultanément plusieurs locataires. De la perspective du locataire, chacun à son installation logicielle dédiée,
+bien qu'en réalité, ils en partagent tous une seule. Tout cela est possible en exécutant le logiciel sur un serveur et en autorisant
+les colocataires à se connecter à celui-ci par le réseau et/ou par une [API](/fr/application-programming-interface.md), se référer à
```

Comment:
Dans la version anglaise, la partie `(also refer to Client-Server Architecture)` est entre parenthèse.
Ensuite, je propose la traduction suivante `(faisant référence à une architecture client-serveur)`.